gcp/api/server.py

```diff
@@ -404,10 +405,14 @@ def page_limit(self) -> int:
 class QueryContext:
   service_context: grpc.ServicerContext
   page_token: ndb.Cursor | None
-  request_start_time: datetime
+  request_cutoff_time: datetime
   # Use a dataclass to copy by reference
   total_responses: ResponsesCount
 
+  def should_break_page(self, response_count: int):
+    return response_count >= self.total_responses.page_limit(
+    ) or self.request_cutoff_time < datetime.now()
+
```

Comment:
(I know this has already been merged but) nit:
The way the brackets on `page_limit()` has been split across lines is yucky.
Also, IMO, `datetime.now() > self.request_cutoff_time` is a bit easier to understand.

Reply:
Yeah... this is made by the yapf linter to try and get it to fit within the max line width limit. Turns out if you add brackets you can split on multiple lines, I'll try that.

Reply:
A good prose docstring would also assist here...